assets/dev/scss/frontend/_global.scss

```diff
@@ -13,6 +13,11 @@
 	// Reset Style for some themes and browsers
 	hyphens: manual;
 
+	a {
+		box-shadow: none;
+		text-decoration: none;
```

Comment:
How can the Theme Style override this rule?

Reply:
It's stronger